client/src/locales/en/messages.po

```diff
@@ -999,11 +995,23 @@ msgstr "This portfolio also had an estimated <0>net {changeType, select, gain {g
 
 #: src/components/ViolationsSummary.tsx:12
 msgid "This portfolio has an average of <0>{0}</0> open HPD violations per residential unit."
-msgstr "This portfolio has an average of <0>{0}</0> open HPD violations per residential unit."
+msgstr "This portfolio has an average of <0>{0}</0> open HPD violations per residential unit.<<<<<<< HEAD"
 
-#: src/components/BigPortfolioWarning.tsx:17
+#: src/components/BigPortfolioWarning.tsx:21
 msgid "This portfolio may consist of several related companies. <0>Learn more</0>"
-msgstr "This portfolio may consist of several related companies. <0>Learn more</0>"
+msgstr "This portfolio may consist of several related companies. <0>Learn more</0>>>>>>>> master"
+
+#: src/containers/NotRegisteredPage.tsx:88
+msgid "This property doesn't have any residential units, so it is not required to register with HPD."
+msgstr "This property doesn't have any residential units, so it is not required to register with HPD."
+
+#: src/containers/NotRegisteredPage.tsx:99
+msgid "This property has fewer than 3 residential units. If the landlord doesn't reside there, it should be registered with HPD."
+msgstr "This property has fewer than 3 residential units. If the landlord doesn't reside there, it should be registered with HPD."
+
+#: src/containers/NotRegisteredPage.tsx:111
+msgid "This property has more than 2 residential units, so it should be registered with HPD!"
+msgstr "This property has more than 2 residential units, so it should be registered with HPD!======="
```

Comment:
Ooops! Looks like this code conflict syntax `=======` somehow got committed to the messages file here...